gum/gumprocess.h

```diff
@@ -98,6 +106,8 @@ GUM_API void gum_module_enumerate_ranges (const gchar * module_name,
 GUM_API GumAddress gum_module_find_base_address (const gchar * module_name);
 GUM_API GumAddress gum_module_find_export_by_name (const gchar * module_name,
     const gchar * symbol_name);
+GUM_API void gum_process_enumerate_malloc_ranges (
+    GumFoundMallocRangeFunc func, gpointer user_data);
```

Comment:
Another one I missed (sorry): would be good to move this after `gum_process_enumerate_ranges`, as the different "modules" are grouped. (And move the implementation correspondingly.)